src/component/archive/ArchiveListContainer.tsx

```diff
@@ -6,7 +7,13 @@ import { useFeatureArchiveApi } from 'hooks/api/actions/useFeatureArchiveApi/use
 import useToast from 'hooks/useToast';
 import { useFeaturesSort } from 'hooks/useFeaturesSort';
 
-export const ArchiveListContainer = () => {
+interface IArchiveListContainer {
+    projectId?: string;
+}
+
+export const ArchiveListContainer: FC<IArchiveListContainer> = ({
```

Comment:
Instead of building this into the existing container, I think it would be better if we separated the two. Less to think about when using the components. So something like `<ProjectArchiveList />` that uses a dedicated hook for this. There might be some duplication of API logic, but less complexity and the components will only do one thing.